DIMS-Core.Database/DIMS-Core.Database/Procedures/DeleteTask.sql

```diff
@@ -0,0 +1,6 @@
+﻿CREATE PROCEDURE [dbo].[DeleteTask]
+    @TaskId INT
+AS
+BEGIN
```

Comment:
Why have you added begin and end in the procedure?

Reply:
I have added BEGIN and END in the procedure to separate procedure's body from another code.
Like in tutorial:
[https://metanit.com/sql/sqlserver/11.1.php](url)